### DISCUSSION TITLE
Renew GitHub Education Pack

### DISCUSSION THREAD

**Discussion by Marcos0230** [community/community#138886] [GitHub Education]
### Hi everyone,

Hi everyone,

This is the third year that I can benefit from GitHub Education Pack but I can't renew with my new school : it just doesn't let me do it and the option is blanked out.
I need to renew it because of the JetBrains License that I need for my studies.

Can someone help me with that ?

Thanks a lot !

**Comment by Valeriucoding** [comment]
If the github student pack doesn't work, you can try applying directly from the jetbrains website [https://www.jetbrains.com/community/education/#students](https://www.jetbrains.com/community/education/#students)
